www/src/pages/de/other-recs.md

```diff
@@ -0,0 +1,163 @@
+
+- [AutoAnimate Homepage](https://auto-animate.formkit.com/)
+- [AutoAnimate GitHub](https://github.com/formkit/auto-animate)
+- [AutoAnimate Component Snippet](https://gist.github.com/hwkr/3fdea5d7f609b98c162e5325637cf3cb)
+
+### Framer Motion
+
+**Für komplexe Animationen mit deklarativem Code**
+
+Framer Motion bietet eine einfache, deklarative Syntax und ermöglicht es dir mit wenig Zeilen an Code komplexe Animationen bis hin zu Gesten zu erstellen.
+
+- [Framer Motion Homepage](https://framer.com/motion)
+- [Framer Motion Documentation](https://www.framer.com/docs/)
+
+## Deployments, Infrastruktur, Datenbanken und CI
+
+### Vercel
+
+**Um deine App zu hosten**
+
+Vercel hat das Hosting von Web Apps zu einem Kinderspiel gemacht. Wir haben unsere App auf Hunderttausende von Nutzern hochskaliert und es gab nie Probleme. Getrieben durch AWS und mit einer viel besseren Benutzeroberfläche.
+
+- [Vercel Homepage](https://vercel.com/)
+- [Create T3 App Vercel deployment guide](/en/deployment/vercel)
+
+### PlanetScale
+
+**Für Datenbanken ohne sich Sorgen machen zu müssen**
+
+PlanetScale ist die beste "serverless Datenbank Plattform" die wir bisher verwendet haben. Wahnsinnige Skalierbarkeit, großartige Entwicklerzufriedenheit und fantastische Preise. Wenn du SQL verwendest (und hoffentlich Prisma) ist es schwer dies zu schlagen.
+
+- [PlanetScale Homepage](https://planetscale.com/)
+
+### Railway
+
+**Um deine Infrastruktur zu hosten**
+
+"Das moderne Heroku". Die einfachste Möglichkeit einen echten Server hochzufahren. Wenn Vercel und PlanetScale nicht ausreichen, ist Railway wahrscheinlich die beste Wahl. Einfaach auf ein GitHub Repo zeigen lassen und loslegen.
+
+- [Railway Homepage](https://railway.app/)
+
+### Upstash
+
+**Für serverless Redis**
+
+Wir lieber Prisma und PlanetScale aber manche Projekt benötigte manchmal eine performantere Lösung. Upstash ermöglicht es die in-memory Performance von Redis in deinem serverless Projekt zu nutzen, ohne sich um die Infrastruktur und Skalierung kümmern zu müssen.
+
+- [Upstash Homepage](https://upstash.com/)
+
+### Pusher
+
+**Für serverless WebSockets**
+
+Wenn WebSockets der Hauptfokus deines Projekts sind, solltest du vielleicht eine traditionellere Backend Lösung wie [Fastify](https://www.fastify.io/) (welche [auch mit tRPC funktioniert!](https://trpc.io/docs/v10/fastify)) in Betracht ziehen. Für das schnelle Hinzufügen von WebSockets zu einer T3 App ist Pusher eine ausgezeichnete Wahl.
+
+- [Pusher Homepage](https://pusher.com/)
+
+### Soketi
+
+Soketi ist eine selbsthostbare, einfache und schnelle Alternative zu Pusher. Es ist vollständig kompatibel mit der Pusher SDK, welches du verwenden kannst um dich mit dem Server zu verbinden. Soketi serverless befindet sich aktuell noch in der Beta.
```

Comment:
Not sure whether welches or welche is appropriate. Welche reads smoother but when I think about it welches makes sense as well

Reply:
I would stick with "welches" as we refer to "it" as "es" :)